### DISCUSSION TITLE
sftp directory transfer doesn't work.

### DISCUSSION THREAD

**Discussion by hasanbagcaci** [electerm/electerm#3640] [General]
Hi,

I am using electerm latest version. I can't directory transfer from my laptop to linux server or from linux to my laptop via sftp but i can for file. Is that normal or bug? Also i tried this transfer with winscp and its successed.

Electerm Version: 1.40.20
Laptop: win 11
Linux server: rocky 9.4

Regards.

**Comment by zxdong262** [comment]
Any error reported? If too many files in the folder or too big would refuse to transfer since it would tar the folder first, if too many files or too big, it would be very slow

**Comment by hasanbagcaci** [comment]
Hi, 

The error is `Remote→Local[error: file not exist]` and the folder size is 4.7M and has 630 files and directory also the transfer isn't start that gives directly this error.

Regards.

**Comment by hasanbagcaci** [comment]
Hi, 

The problem is going on latest version. Also i reinstall windows 11. I send ss about problem below.

![Screenshot_of_sftp_error](https://github.com/user-attachments/assets/2f89f7db-628f-43af-b9ed-7947deac3c2b)

**Comment by zxdong262** [comment]
does the user have permission on /var?

**Comment by hasanbagcaci** [comment]
Yes,  it's root user.

**Comment by zxdong262** [comment]
Can you test transfer a common folder, no system file in it? I suspect it is because some files are link files

**Comment by hasanbagcaci** [comment]
I created emty folder but problem is going on.

**Comment by zxdong262** [comment]
Can you try download to another folder ,not onedrive, might be the virtual folder's issue?

**Comment by hasanbagcaci** [comment]
I created folder under the c:\ as  a empty folder but i got same error again.

![Screenshot_of_sftp_error2](https://github.com/user-attachments/assets/e2c3e0db-e826-4153-8ca1-cb5cc4372116)

**Reply by zxdong262** [reply]
can you try use another user other than root to have a test?

**Comment by hasanbagcaci** [comment]
It is ok. I created new user and new folder in user's home folder as an empty and the transter was successful. Also I tried  this transfer to desktop folder insead of under the c:\ it was successful. 

I want to say about i am doing this transfer with root and full folder  but too rare. Generally i am getting error.
